internal/internal.go

```diff
@@ -224,17 +223,17 @@ func RunSyncSet(logger *log.Logger, source Source, destination Destination, conf
 
 	results := destination.ApplyChangeSet(changeSet, eventLog)
 
-	logger.Printf("Sync results: %v users added, %v users updated, %v users removed\n",
-		results.Created, results.Updated, results.Deleted)
-
-	for i := 0; i < 100; i++ {
+	for range 100 {
```

Comment:
But the for loop style was more readable for those less familiar with go. :)

Reply:
You have a point. Makes me wonder how many different styles of `for` loop exist among major programming languages. 🤔